android/src/main/java/com/mybigday/rnmediameta/RNMediaMeta.java

```diff
@@ -78,7 +79,18 @@ private void putString(WritableMap map, String key, String value) {
     if (value != null) map.putString(key, value);
   }
 
-  private void getMetadata(String path, Promise promise) {
+  private void getMetadata(String pathOrJSONString, Promise promise) {
```

Comment:
I'd recommend to use [`ReadableMap`](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableMap.java) make `options` arg, just like:

```java
private void getMetadata(String path, ReadableMap options, Promise promise`) { ... }
```

it will also need to change `get` react method.